client/components/App.jsx

```diff
@@ -39,6 +40,26 @@ class App extends Component {
     zoomToken: '',
   };
 
+  static getInterviewDetails = (description) => {
+    const FIRST_IDX = 5;
+    const LAST_IDX = 6;
+    const EMAIL_IDX = 7;
+    const CHAT_IDX = 3;
+
+    // Determine whether the interview is based on the newer format
+    const isNewer = description.indexOf(interviewsNewMetadata) >= 0;
+
+    if (isNewer) {
+      const soonestInterviewDetails = description.split('\n');
+      const parseResults = description.split('\n').map(el => el.split(': ')[1]);
+      return [`${parseResults[FIRST_IDX]} ${parseResults[LAST_IDX]}`,
```

Comment:
This would be easier to read and to match up to the usage below if you assigned these as variables first and did `return [candidateName, candidateEmail, tlkioLink]`. That would also obviate the need for assigning those index constants above, since it's pretty obvious that `const candidateName = parsedResults[5] + ' ' + parsedResults[6]` is hardcoding the indices.

Reply:
I agree, I tend to minimize the amount of magic numbers/strings I would have to use, so I decided to put them inside constants, but assigning to variables would be better here.